new-frontiers-14/frontier-station-14
# More Infinite Supply Vendors

## About the PR



Makes more vendors infinite supply (LessLethalVend, LibertyVend, CiviMed, SalvageVend, most of the AstroVend).  Removes CiviMed resupply, removes vendor resupplies for infinite vendors from dungeons.

~~Changes the value of welding fuel, plasma and uranium in the FuelVend to 750, 3000, and 3500 (the latter two _must_ be more expensive than the lowest price available in the cargo market).~~  Moved elsewhere.

The CiviMed becomes, like the FuelVend, a snarky jerk with high prices.  The goal here is to establish a reasonable baseline for medical services while allowing medical supply to exist when doctors are not around - it will be expensive, and it will suck, but it's there.  The CiviMed receives a wallmount version, and is mapped to replace the existing NanoMed and NanoMed Plus vendors on POIs and stations.  All shuttle vendors have been left alone, though a restock was removed from the Bison (as the salvage vendor itself is now infinite).

I'm putting this up as a proposal, I think the CiviMed prices may need review, and certain items in each inventory may need review, but the concept itself should be feature complete and warrants some discussion, just trying to get the ball rolling.

Full disclosure, I've also pork barreled a small syndicate grammar fix in here.

## Why / Balance


Low pop hours need some source of medical supply, and without restocks, it's difficult to get without buying a ship (which we should not want) or going on expedition.

## How to test


1. Buy things from the CiviMed, peruse the prices.  The sprites should make sense compared to the wallmount NanoMed.
2. Compare the contents of the NanoMed vs. the CiviMed, check the contents of the Salvage vendor, AstroVend, YouTool, EngiVend, Vendomat, LibertyVend, and LessLethalVend.  They should be mostly infinite on POIs.

## Media


Comparison of the NanoMed vs. the CiviMed inventories: note that the CiviMed's stock is infinite.  The NanoMed as-is is being largely repurposed for doctor supplies on medical ships.
![image](https://github.com/user-attachments/assets/aaf2f12e-fb1d-4eed-8dcd-90dd970eca28)


## Breaking changes


**Changelog**



:cl:
- tweak: Most POI vendors (save drobes) have an infinite stock of items.
